src/cmd/ksh93/tests/io.sh

```diff
@@ -588,3 +588,11 @@ n=$( exec {n}< /dev/null; print -r -- $n)
 
 # According to POSIX <> should redirect stdin when no fd is specified
 read -n 1 -t 0.1 <>/dev/zero || log_error '<> should redirect stdin by default'
+
+TMPF=$(mktemp -p $TEST_DIR)
+echo foo >$TMPF
+export TMPF
+
+[ -n "$($SHELL -c 'echo $(<$TMPF)' <&-)" ] || log_error "Closing stdin causes failure when reading file through \$(<)"
```

Comment:
Any reason you're not using the `[[ ... ]]` syntax?

Reply:
Fixed.